virt/virt-2-6-release-notes.adoc

```diff
@@ -65,6 +65,7 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 === Web console
 
 //CNV-9312: The VM wizard has been completely redesigned, simplifying VM creation
+The virtual machine wizard has been redesigned to simplify the process of xref:../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[creating virtual machines] by using preconfigured xref:../virt/vm_templates/virt-creating-vm-template.adoc#virt-creating-vm-template[virtual machine templates].
```

Comment:
Oh, much leaner!
Can you please add a leading bullet point and I'll be happy to merge this

Reply:
Bullet it is! @aburdenthehand - this one is ready :-)